airflow/contrib/hooks/sftp_hook.py

```diff
@@ -214,3 +214,12 @@ def get_mod_time(self, path):
         conn = self.get_conn()
         ftp_mdtm = conn.stat(path).st_mtime
         return datetime.datetime.fromtimestamp(ftp_mdtm).strftime('%Y%m%d%H%M%S')
+
+    def path_exists(self, path):
+        """
+        Returns True if a remote entity exists
+        :param path: full path to the remote file or directory
```

Comment:
```suggestion

                :param path: full path to the remote file or directory
```
New line is required in this place. Otherwise, docs doesn't looks good.
<img width="688" alt="Screenshot 2019-10-19 at 01 52 38" src="https://user-images.githubusercontent.com/12058428/67134612-4a845e00-f213-11e9-96a2-b83efd76a7da.png">

Reply:
Thanks, I've applied your suggestion.